docs/modules/ROOT/nav.adoc

```diff
@@ -5,7 +5,6 @@
 * Runtime Descriptions
 ** xref:runtime/xcm_executor.adoc[XCM Executor]
 * Pallet Specifications
-** xref:pallets/aura_ext.adoc[cumulus_aura_ext]
```

Comment:
why is this removed?

Reply:
This is a duplicate of the same link 2 lines lower